components/cas-inbound-authenticator/src/main/java/org/wso2/carbon/identity/sso/cas/util/CASSSOUtil.java

```diff
@@ -255,96 +250,36 @@ public static TicketGrantingTicket getTicketGrantingTicket(String ticketGranting
                 mappedClaims.add(claimMapping.getLocalClaim().getClaimUri());
             }
 
-            // Get all supported claims
-            ClaimManager claimManager = userRealm.getClaimManager();
-            org.wso2.carbon.user.api.ClaimMapping[] mappings = claimManager.getAllClaimMappings();
-            for (org.wso2.carbon.user.api.ClaimMapping claimMapping : mappings) {
-                allClaims.add(claimMapping.getClaim().getClaimUri());
-                if (log.isDebugEnabled()) {
-                    log.debug("adding requested claim: " + claimMapping.getClaim().getClaimUri());
-                }
-            }
-
-            // Get claim values for the user
-            UserStoreManager userStoreManager = null;
-            boolean localAuthentication = false;
-            try {
-                userStoreManager = userRealm.getUserStoreManager();
-                tenantAwareUsername = MultitenantUtils.getTenantAwareUsername(username);
-                if (log.isDebugEnabled()) {
-                    log.debug("getUserClaimValues: username=" + tenantAwareUsername);
-                }
-                localAuthentication = userStoreManager.isExistingUser(tenantAwareUsername);
-            } catch (UserStoreException e) {
-                // User came from federated authentication
-                log.warn("Error while retrieving the user from federated authentication, e");
-            }
-
-            Map<String, String> localClaimValues = new HashMap<String, String>();
-
-            if (userStoreManager == null || !localAuthentication) {
-                Map<ClaimMapping, String> userAttributes = result.getSubject().getUserAttributes();
-                if (userAttributes != null) {
-                    for (Map.Entry<ClaimMapping, String> entry : userAttributes.entrySet()) {
-                        if (log.isDebugEnabled()) {
-                            log.debug(entry.getKey().getLocalClaim().getClaimUri() + " ==> " + entry.getValue());
-                        }
-                        claimsMap.put(entry.getKey().getLocalClaim().getClaimUri(), entry.getValue());
+            Map<ClaimMapping, String> userAttributes = result.getSubject().getUserAttributes();
```

Comment:
Shall we add a null check for result and getSubject as well

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-cas/pull/21/commits/278dbbb011dddf4ff853c8a9a2810b26cd48c187